ratatool-scalacheck/src/main/scala/com/spotify/ratatool/scalacheck/HashMapBuildable.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.spotify.ratatool.scalacheck
+
+import org.scalacheck.util.Buildable
+
+private[scalacheck] object HashMapBuildable {
```

Comment:
Trying to port that upstream in https://github.com/typelevel/scalacheck/pull/1023